test/Microsoft.ML.StaticPipelineTesting/Training.cs

```diff
@@ -169,8 +168,7 @@ public void SdcaBinaryClassificationNoCalibration()
             var est = reader.MakeNewEstimator()
                 .Append(r => (r.label, preds: ctx.Trainers.Sdca(r.label, r.features,
                 maxIterations: 2,
-                loss: loss, onFit: p => pred = p,
-                advancedSettings: s => s.NumThreads = 1)));
```

Comment:
> s.NumThreads = 1 [](start = 38, length = 17)

Here too.

Reply:
fixed

---
In reply to: [248104923](https://github.com/dotnet/machinelearning/pull/2093#discussion_r248104923) [](ancestors = 248104923)